ds-controller-board/driver/service.go

```diff
@@ -0,0 +1,50 @@
+package driver
+
+import (
+	"github.com/edgexfoundry/device-sdk-go/v2/pkg/service"
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/clients/logger"
+	contract "github.com/edgexfoundry/go-mod-core-contracts/v2/models"
+	"github.com/pkg/errors"
+)
+
+// ServiceWrapper wraps an EdgeX SDK service so it can be easily mocked in tests.
+type ServiceWrapper interface {
```

Comment:
Why add this?

Reply:
the intefaces module is not in the device SDK 2.2.0 version

Reply:
I needed the load custom config. thats why I added the Service wrapper following how device LLRP was before I removed it in Edgex for 2.3

Reply:
load custom config is available without this. Only reason to add this is so you can mock the API in unit tests, which we are not adding. LoadCustomConfig is available from object return by `service.RunningService()`